dali/pipeline/operators/util/optical_flow.cc

```diff
@@ -40,36 +40,75 @@ For values <=0, grid size is undefined. Currently only grid_size=4 is supported.
 
 DALI_REGISTER_OPERATOR(OpticalFlow, OpticalFlow<GPUBackend>, GPU);
 
+constexpr int kNOutputDims = 2;
+constexpr int kNInputDims = 4;
+
 
 template<>
 void OpticalFlow<GPUBackend>::RunImpl(Workspace<GPUBackend> *ws, const int) {
   if (enable_hints_) {
+    DALI_ENFORCE(false, "Hints currently ignored");
+    // TODO(mszolucha): hints currently ignored, add feature
     const auto &input = ws->Input<GPUBackend>(0);
     const auto &external_hints = ws->Input<GPUBackend>(1);
     auto &output = ws->Output<GPUBackend>(0);
+    size_t nsequences_ = input.ntensor();
+
+    of_lazy_init(frames_width_, frames_height_, depth_, ws->stream());
+
+    output.Resize(
+            {nsequences_, {2, (frames_height_ + 3) / 4, (frames_width_ + 3) / 4, kNOutputDims}});
```

Comment:
Why do you uncoditionally divide the size by 4, and not based on options? Can't the OF output flow with the same size as image? 

Is the output presented in planes as this `2` suggests? If so, why?

Reply:
On the second thought, it might be wise to ask OF "kernel" implementation, what will be the output size based on the input image size.

Reply:
Will do that in next PR